rfcs/061-content-api-next-steps/README.md

```diff
@@ -0,0 +1,312 @@
+# Content-api: next steps
+
+[Background information](#background-information)  
+[Challenges encountered and decisions made](#challenges-encountered-and-decisions-made)  
+  - Scheduled events  
+  - Delist  
+  - Sorting and filter by date  
+  
+[Proposal](#proposal)  
+- Events endpoint and response  
+- Event endpoint and response  
+- Filters  
+- Aggregations  
+- Sorting  
+
+[Further requests for opinions and insights](#further-requests-for-opinions-and-insights  )  
+- Prismic IDs  
+- How granular do we want to be for filtering by time and date on Events?
+
+
+## Background information
+Around October 2023, we started work on adding our Prismic Events to the Content API ([see reasoning here for why we need the Content API](https://github.com/wellcomecollection/docs/tree/main/rfcs/055-prismic-api#background)). We did this without writing an RFC first, opting for a few chats and relying on the [Original suggestion for Events](https://github.com/wellcomecollection/docs/blob/main/rfcs/055-prismic-api/events.md) that was written at the start of building the Content API. We realised this should not have been the case, and are now choosing to write one for a few reasons; to document the challenges we’ve come across, the decisions we've made along the way and how we propose it changes and ends up looking like.
+
+The active endpoints are:
+- https://api.wellcomecollection.org/content/v0/events
+- https://api.wellcomecollection.org/content/v0/events/[eventId]
+
+Other references:  
+[Original suggestions for future versions of Content API](https://github.com/wellcomecollection/docs/blob/main/rfcs/055-prismic-api/future-versions.md) (includes suggestions of filters and such for events)
+
+Notes:  
+- This document will be using the term “Scheduled” to talk about “children” events.
+- This API only serves Prismic `Events` content types. Please note that it doesn't fetch anything from the `Events Series` content type. The name of a series does get considered at the display level as it gets displayed on the result card along with a “Part of” prefix, but gets pulled from the `Event` data.
+- Some of these challenges also affect the `Articles` endpoint. If that is the case, it will be indicated.
+
+
+## Challenges encountered and decisions made  
+
+### Scheduled events
+This [Github PR](https://github.com/wellcomecollection/content-api/issues/93#issue-2088674788) described this and applied a solution that is subject to change.
+
+#### Challenge
+There is no way to create an event hierarchy at the moment. In reality, some events come as a set (they become “Scheduled” events) and could use such a thing. The solution adopted by the team is to create a “Parent” event first, which contains the general information including promotional images. Then, “Scheduled” events are added to its “Scheduled” tab:
+
+// place image here 
```

Comment:
I know this is WIP but just in case, images are missing!

Reply:
Added